core/src/either.rs

```diff
@@ -1,4 +1,21 @@
+use std::ops::Deref;
+
 pub enum Either<A, B> {
     Left(A),
     Right(B),
 }
+
+impl<A, B> Deref for Either<A, B>
```

Comment:
Is this used anywhere?

Reply:
It's used [here](https://github.com/ruffle-rs/ruffle/pull/7564/files#diff-2da3faadf637b2f2f7092be0c2d89acfd095aa771b58721df2113b2dd062faebR196)